static/css/styleguide-override.css

```diff
@@ -0,0 +1,199 @@
+/*
+  NOTE:
+    - Here exceptions,updates, missing or on-going style in styleguide.neomad.org
+    - This file should be empty... and remove soon
+
+  ENHANCEMENT:
+    - header.main should be body > header removing the class
+    - footer should be body > footer to scope style deeper
+    - define different class for body > main to manage padding rules
+    - define alert style
+*/
```

Comment:
I'm not reviewing this as it's meant to be removed

Reply:
👍